tests/monitoring/metrics.go

```diff
@@ -61,10 +61,11 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 		var excludedMetrics = map[string]bool{
 			// virt-api
 			// can later be added in pre-existing feature tests
-			"kubevirt_portforward_active_tunnels":  true,
-			"kubevirt_usbredir_active_connections": true,
-			"kubevirt_vnc_active_connections":      true,
-			"kubevirt_console_active_connections":  true,
+			"kubevirt_portforward_active_tunnels":            true,
+			"kubevirt_usbredir_active_connections":           true,
+			"kubevirt_vnc_active_connections":                true,
+			"kubevirt_console_active_connections":            true,
+			"kubevirt_vmi_last_connection_timestamp_seconds": true,
```

Comment:
instead of ignoring can we move the change https://github.com/kubevirt/kubevirt/pull/11934/files#diff-01bea105c8094d4b6a3eaa31470511407db6dd687d6bec6967dfe435e1b2cde4R278 to the `setupVM`?

and we can still do `It("should have kubevirt_vmi_last_connection_timestamp_seconds correctly configured", func() {` inside this context, and we would reduce the number of VMs we would be creating

Reply:
I'd like to keep it as it. 
Firstly, it won't reduce the number of VMs we would be creating, as long as we add the `It("should have kubevirt_vmi_last_connection_timestamp_seconds correctly configured", func() {`, since we add another execution of the `BeforeEach()`. Secondly, this change requires changing `setupVM()` to return the VMI (the new test requires its name), and `createRunningVM()` to create other type of VM, and not guestless which would create heavier VM for other test that don't need it.

Reply:
per https://onsi.github.io/ginkgo/#mental-model-how-ginkgo-traverses-the-spec-hierarchy

wouldn't the `BeforeEach` just run before the all `Context("Prometheus metrics"`?

Reply:
could we use https://onsi.github.io/ginkgo/#setup-in-ordered-containers-beforeall-and-afterall, and setup a single VM for all the tests?

Reply:
I'm not sure. There is no use in whole repo, and I found https://github.com/kubevirt/kubevirt/blob/63bc1132a5b36dfbd774ee37a7e5335f62ffffc3/tests/operator/operator.go#L3438.

Reply:
@victortoso Do you happen to know what's our take on `BeforeAll()`?

Reply:
Actually, no. Seems that we should use only in very specific cases (e.g: Needs to be labeled as Serial as per [tests/README.md](https://github.com/kubevirt/kubevirt/blob/main/tests/README.md)). My grep shows no use of it which means some discussion have had happened in the past. Found this one https://github.com/kubevirt/kubevirt/issues/6123 where brings the argument of `is keeping the test suite from evolving properly` so I'd avoid it.

Reply:
@machadovilaca So another VMI will be created anyway, if we add `It("should have kubevirt_vmi_last_connection_timestamp_seconds correctly configured", func() {`.. I don't see a reason to complicate the existing `setupVM()` and `createRunningVM()`, but let me know if you don't agree.

Reply:
yes, then makes sense to keep this one guestless